spec/std/file_spec.cr

```diff
@@ -582,41 +582,38 @@ describe "File" do
       File.expand_path("~/a", "/tmp/gumby/ddd").should eq(File.join([home, "a"]))
     end
 
-    # TODO: these specs don't compile on windows because ENV isn't ported
     # TODO: remove /\A\/\// hack after this is removed from macros
```

Comment:
please check the diff for #5623 and do this TODO too please

Reply:
Sorry, I'm not sure what you mean... reinstating `%r{\A//}`?

Reply:
yes